server.js

```diff
@@ -6,7 +6,7 @@ var app = express();
 var server = http.createServer(app);
 var io = require('socket.io').listen(server);
 
-var bug = {};
+var bug = {livetext: "Live", loctext: "York University"};
```

Comment:
Doesn't seem right hard coding the default location to be `York University` as although correct for this year isn't generic. (Yes I know its an easy thing to change :) )